dependencies/Makefile.mac

```diff
@@ -7,16 +7,14 @@ DEPENDENCIES_URL = https://cyberbotics.com/files/repository/dependencies/mac/rel
 WGET = LANG=en_US.UTF-8 wget -qq
 
 QT_PACKAGE = qt-6.2.3-release.tar.bz2
-ASSIMP_PACKAGE = assimp-5.2.3.zip
-FREETYPE_PACKAGE = macos-precompiled-freetype-2.9.tar.bz2
+ASSIMP_PACKAGE = assimp-5.2.3.tar.bz2
 FFMPEG_PACKAGE = ffmpeg.tar.bz2
-LUA_PACKAGE = lua-5.2.3.tar.bz2
-LUA_GD_PACKAGE = gd.tar.bz2
```

Comment:
lua gd dependency has been removed on macOS and Windows but on Linux is still downloaded.
Don't we want to remove it as well?